README.md

```diff
@@ -41,6 +41,10 @@ For quick-and-easy satisfaction of this requirement, nprintML supplies the boots
 
 nPrint may thereby be installed system-globally, to the user environment, to the (virtual) environment to which nprintML was installed, or to a specified path prefix. Consult the command's `--help` for more information.
 
+`nprint-install` is identically available through its Python module (no different from `pip` above):
+
+    python -m nprintml.net.install
```

Comment:
(I figured that, again for debugging the system configuration of Python, we may as well make this CLI submodule executable as well.)